docs/src/PageHeader.doc.js

```diff
@@ -0,0 +1,645 @@
+        selected.filter((selectedItem) => selectedItem.value != item.value),
+      );
+    } else {
+      setSelected((selected) => [...selected, item]);
+    }
+  };
+
+  return (
+    <Flex gap={8} direction="column" flex="grow">
+      <PageHeader
+        title="Product groups"
+        subtext="S. E. All products USD"
+        primaryAction={<Button color="red" text="Create product group" />}
+        secondaryAction={<Button text="Promote" />}
+      />
+      <Divider />
+      <PageHeader
+        title="Custom reports"
+        primaryAction={<Button color="red" text="Create new report" />}
+        secondaryAction={
+          <React.Fragment>
+            <Tooltip idealDirection="up" text="Board options">
+              <IconButton
+                accessibilityControls="page-header-example"
+                accessibilityHaspopup
+                accessibilityExpanded={open}
+                accessibilityLabel=""
+                icon="ellipsis"
+                iconColor="darkGray"
+                selected={open}
+                onClick={() => setOpen((prevVal) => !prevVal)}
+                ref={anchorRef}
+              />
+            </Tooltip>
+            {open && (
+              <Dropdown
+                id="page-header-example"
+                anchor={anchorRef.current}
+                onDismiss={() => {
+                  setOpen(false);
+                }}
+              >
+                <Dropdown.Item
+                  handleSelect={handleSelect}
+                  selected={selected}
+                  option={{
+                    value: 'Share Report',
+                    label: 'Share Report',
+                  }}
+                />
+                <Dropdown.Item
+                  handleSelect={handleSelect}
+                  selected={selected}
+                  option={{ value: 'Help center', label: 'Help center' }}
+                />
+                <Dropdown.Item
+                  handleSelect={handleSelect}
+                  selected={selected}
+                  badgeText="New"
+                  option={{
+                    value: 'Delete',
+                    label: 'Delete',
+                  }}
+                />
+              </Dropdown>
+            )}
+          </React.Fragment>
+        }
+      />
+    </Flex>
+  );
+}
+        `}
+      />
+    </MainSection.Subsection>
+
+    <MainSection.Subsection
+      description={`\`subtext\` should be used to add metadata about the content on the page, not to describe the page itself.`}
+      title="Subtext"
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
+<PageHeader
+  title="Create product group"
+  subtext="2,131 products"
+  primaryAction={<Button color="red" text="Quick create" />}
+/>;
+        `}
+      />
+    </MainSection.Subsection>
+
+    <MainSection.Subsection
+      description={`\`maxWidth\` should be set when the content of the page is centered and has a maximum width itself.`}
```

Comment:
I'm not sure if this is visually communicating what we're trying to get across? Would a contrasting background help demonstrate the page max width?
<img width="844" alt="Screen Shot 2021-03-12 at 1 18 56 PM" src="https://user-images.githubusercontent.com/12059539/110999689-9e767200-8335-11eb-8ca3-c384b67f356c.png">